base_location_geonames_import/__manifest__.py

```diff
@@ -9,22 +8,28 @@
 
 {
     'name': 'Base Location Geonames Import',
-    'version': '11.0.1.1.1',
+    'version': '12.0.1.1.1',
     'category': 'Partner Management',
     'license': 'AGPL-3',
-    'summary': 'Import better zip entries from Geonames',
+    'summary': 'Import zip entries from Geonames',
     'author': 'Akretion,'
               'Agile Business Group,'
-              'Antiun Ingeniería S.L.,'
+              u'Antiun Ingeniería S.L.,'
```

Comment:
You can remove this one, as it's the "old Tecnativa"

Reply:
Seeing that you have added the `u` prefix, you have installed maintainer-tools with Python 2 instead with Python 3, and that's why you need this.